docs/platform/concepts/service_backups.md

```diff
@@ -319,3 +343,80 @@ backups, see
+the **Configure backup settings** section on your <ConsoleLabel name="Backups"/> page
+in the [Aiven Console](https://console.aiven.io).
+
+- **Enabled (`dfs`)**: When you set `dragonfly_persistence` to `dfs`, Aiven for Dragonfly
+  performs DFS dumps every 10 minutes whenever a key changes. These dumps provide
+  additional data protection against Dragonfly service incidents, limiting potential data
+  loss to a maximum of 10 minutes. However, full backups are created only according to
+  the backup schedule (every 12 hours) or when the service is shut down.
+
+- **Disabled (`off`)**: When you set `dragonfly_persistence` to `off`, Aiven for Dragonfly
+  stops all DFS dumps and backups. If the service restarts or powers off for
+  any reason, you may lose any data not yet backed up. Additionally, you cannot
+  fork or replicate the service, which can affect potential scaling or disaster recovery
+  plans.
+
+  :::warning
+  If you disable `dragonfly_persistence`, the system immediately deletes all existing
+  backups, preventing any data recovery from those backups. When you re-enable
+  persistence, it starts a new backup cycle, but it won't restore any previously
+  stored data.
+  :::
+
+:::note
+The Append Only File (AOF) persistence method is not supported for the managed
+Aiven for Dragonfly service.
+:::
+
+### Aiven for Valkey™
+
+Aiven for Valkey automatically backs up data every 12 hours and supports configurable
+data persistence using Redis Database Backup (RDB).
+
+#### Persistence settings
+
+You can configure the `valkey_persistence` settings from <ConsoleLabel name="actions"/> >
+the **Configure backup settings** section on your <ConsoleLabel name="Backups"/> page
+in the [Aiven Console](https://console.aiven.io).
+
+- **Enabled (`rdb`)**: When you set `valkey_persistence` to `rdb`, Aiven for Valkey
+  performs RDB dumps every 10 minutes whenever a key changes. These dumps provide
+  additional data protection against Valkey service incidents, limiting potential data
+  loss to a maximum of 10 minutes. However, full backups are created only according to
+  the backup schedule (every 12 hours) or when the service is shut down.
+- **Disabled (`off`)**: When you set `valkey_persistence` to `off`, Aiven for Valkey
+  stops all RDB dumps and backups. If the service restarts or powers off for
+  any reason, you may lose any data not yet backed up. Additionally, you cannot
+  fork or replicate the service, which can affect potential scaling or disaster recovery
+  plans.
+
+  :::warning
+  If you disable `valkey_persistence`, the system immediately deletes all existing
+  backups, preventing any data recovery from those backups. When you re-enable
+  persistence, it starts a new backup cycle, but it won't restore any previously
+  stored data.
+  :::
+
+:::note
+The Append Only File (AOF) persistence method is not supported for the managed
+Aiven for Valkey service.
+:::
+
+#### Edit the backup schedule
+
+<EditBackUpSchedule/>
```

Comment:
The steps to edit a backup are incorrect. The current instructions state:
- Click Service settings page.
- In Advanced configuration, click Configure.

However, the correct steps are:
- Click Backup in the sidebar.
- Click Actions (...) > Configure backup settings. See the attached screenshot.
 
![image](https://github.com/user-attachments/assets/be9b43fa-c97c-4806-838a-faee5eb34961)